src/classes/BDI_DataImport_BATCH.cls

```diff
@@ -239,8 +239,20 @@ public with sharing class BDI_DataImport_BATCH implements Database.Batchable<sOb
 	        	list<string> strParts = strHelpText.split('\\.', 2);
 	        	if (strParts.size() == 2 && strParts[0] == strObj && strParts[1] != '') {
 	        		string strField = strParts[1];
-	        		if (!UTIL_Namespace.InManagedPackage)
-	        		     strField = UTIL_Namespace.StrTokenRemoveNSPrefix(strField);
+	        		//if we aren't in the npsp namespace, remove those tokens from help text
+                    //also removes tokens from actual detected namespace, if we happen to be
+                    //in a non-npsp namespace
+                    if (UTIL_Namespace.getNamespace() != 'npsp') {
+                        string strFieldOld = strField;
+                        strField = UTIL_Namespace.StrTokenRemoveNSPrefix(strField);
+                        
+                        //if we have a detected namespace, and removed a namespace from the current field
+                        //add the current namespace back in, allowing this code to run in any namespace
+                        if (strField != strFieldOld && UTIL_Namespace.getNamespace() != '') {
+                            strField = UTIL_Namespace.StrAllNSPrefix(strField);
```

Comment:
We're removing the npsp__ namespace from the help text if we're not in the npsp namespaced package. Then if we're in another namespace, we're adding the namespace back to the ones we just removed npsp__ from so the field describes are handled correctly.